File: app/src/main/java/com/battlelancer/seriesguide/jobs/episodes/ShowBaseJob.java
package com.battlelancer.seriesguide.jobs.episodes;

public abstract class ShowBaseJob extends BaseEpisodesJob {

    private final long showId;

    public ShowBaseJob(long showId, int flagValue, JobAction action) {
        super(flagValue, action);
        this.showId = showId;
    }

    @Override
    public long getShowId() {
        return showId;
    }
}

File: app/src/main/java/com/battlelancer/seriesguide/jobs/episodes/ShowBaseJob.kt
// SPDX-License-Identifier: GPL-3.0-or-later
// Copyright 2017, 2018, 2021, 2023, 2024 Uwe Trottmann
package com.battlelancer.seriesguide.jobs.episodes

abstract class ShowBaseJob(
    override val showId: Long,
    flagValue: Int,
    action: JobAction
) : BaseEpisodesJob(flagValue, action)
